### DISCUSSION TITLE
How to enable auto deployment on one project but disable on others two (which uses same repo)

### DISCUSSION THREAD

**Discussion by adarshaacharya** [vercel/community#6316] [Help]
### Question

I'm using same master branch to make the deployment for three deployment of the same repo. (i.e 3 projects of same repo)
I've disabled auto deployment on merging to master branch.

Here's my `vercel.json` file:

```
{
  "framework": "nextjs",
  "git": {
    "deploymentEnabled": {
      "master": false
    }
  }
}

```

Requirement:
I want to enable auto deployment on `projectA` but disable on `projectB` and `projectC` respectively.

Is there any way to achieve above requirements

**Comment by amyegan** [comment]
Hey @adarshaacharya. You can use Ignored Build Step to stop builds on `projectB` and `projectC`.

https://vercel.com/docs/projects/overview#ignored-build-step
